doc/cabal-commands.rst

```diff
@@ -36,6 +35,7 @@ Commands
       freeze                 Freeze dependencies.
       gen-bounds             Generate dependency bounds.
       outdated               Check for outdated dependencies.
+      path                   Query for simple project information
```

Comment:
To be consistent with the rest, it has to end with a full stop.
```suggestion
      path                   Query for simple project information.
```

Reply:
I grabbed the `cabal help` output verbatim. I'm happy to fix this problem as well.

Reply:
Thanks! Yes, this is an unfortunate divergence, but if you look around the RST file, the descriptions use the full stop. We definitely want consistency inside one source, at least. It would be great to make it consistent across the sources (RST and help) one day.

Reply:
We could run `cabal help`, save that to a file and then snippet it into the RST docs. I'm doing this for haddock with [haddock-docs: Warn about separating lists, use snippets](https://gitlab.haskell.org/ghc/ghc/-/merge_requests/13205).

Reply:
Do mean run it once manually? It is an option but not a great one. A great option would be something automated. But I don't expect it to be completely trivial: the RST has more stuff than help (iirc), so it'd require some kind of factoring.

Also, if we do the sync (one way or another), I'd prefer full stops to stay: this makes more sense to me typographically.

Reply:
In the ghc/haddock merge request I linked, there's a makefile that does the snippet generation, that generates `*.html` and `*.tex` snippets from a source `*.hs` file.

Reply:
Yeah, this looks very cool. Would be great to integrate this into Cabal I think.